packages/lexical-rich-text/src/index.ts

```diff
@@ -983,7 +989,12 @@ export function registerRichText(editor: LexicalEditor): () => void {
     editor.registerCommand(
       COPY_COMMAND,
       (event) => {
-        copyToClipboard(editor, event instanceof ClipboardEvent ? event : null);
+        copyToClipboard(
+          editor,
+          event instanceof getClassFromEditorWindow(editor, ClipboardEvent)
+            ? event
+            : null,
```

Comment:
```suggestion
	 Object.getPrototypeOf(event).constructor.name === ClipboardEvent.name ? event : null,
```

where ClipboardEvent.name would be hoisted

Given that we're also using this in the Clipboard module we can also abstract this on a clipboard exported function.

Reply:
That's a good idea! I updated the PR using the new approach and not relying on the window object. I've created a util function that at least makes TypeScript happy about it.